packages/zoe/test/unitTests/contractSupport/test-zoeHelpers.js

```diff
@@ -859,3 +596,99 @@ test('ZoeHelpers makeEmptyOffer', async t => {
     t.assert(false, e);
   }
 });
+
+test('ZoeHelpers isOfferSafe', t => {
+  t.plan(5);
+  const { moolaR, simoleanR, moola, simoleans, amountMaths } = setup();
+  const leftOfferHandle = harden({});
+  const rightOfferHandle = harden({});
+  const cantTradeRightOfferHandle = harden({});
+  const reallocatedHandles = [];
+  const reallocatedAmountObjs = [];
+  const completedHandles = [];
+  try {
+    const mockZCF = harden({
```

Comment:
I don't see anything here that wouldn't work with the mock builder. Why did you prefer to build manually?

(If it's worse in some way, we should pull the whole thing out rather than having multiple ways to build the mock.)

Reply:
I didn't know the mock builder existed :)